unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/Tests/GLTFShape_Tests.cs

```diff
@@ -34,6 +35,56 @@ public IEnumerator ShapeUpdate()
             "'GLTFScene' child object with 'InstantiatedGLTF' component should exist if the GLTF was loaded correctly");
     }
 
+    [Test]
+    public void CustomContentProvider()
+    {
+        string entityId = "1";
+        TestHelpers.CreateSceneEntity(scene, entityId);
+
+        Assert.IsTrue(
+            scene.entities[entityId].gameObject.GetComponentInChildren<UnityGLTF.InstantiatedGLTFObject>() == null,
+            "Since the shape hasn't been updated yet, the 'GLTFScene' child object shouldn't exist");
```

Comment:
If we are going to test this, it should be another test. Let's make them as unitary as possible.